### DISCUSSION TITLE
cn/2024/11/18/sm/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#187] [Announcements]
# cn/2024/11/18/sm/

https://hongtaoh.com/cn/2024/11/18/sm/

**Comment by wangyunzi** [comment]
可能这个每个人的算法不一样了，我每天打开小红书就是搜索各种做菜的教程，把这个用成了一个工具软件还是蛮好用的，有时候也会搜索一下生产中需要的常识之类的😉

**Reply by hjz2016** [reply]
你一打开xhs，发现别人做的菜都是光鲜亮丽如同大厨出品，你是否会认为大部分人做菜都比自己强？这算不算一种扭曲呢。

**Reply by wangyunzi** [reply]
不会啊，这属于一种极端想法，我只会觉得别人很厉害，但是我相信自己也能做到这样的水平，而且人家做出的是属于人家的，我只想要的是做菜技能，不会想那么多。如果你自己有自己想要的东西，想要达到的目的，这些东西都只能说是手段而已，如果我仅仅因为在互联网上看到别人那么好而否定自己，那我也太脆弱不堪一击了吧，属于没有自己稳定的内核的人，那就少看这些。

**Reply by hongtaoh** [reply]
@wangyunzi 你真的是幸运。我发现确实很多人可以完全不受这些社交媒体影响。很可惜的是，我不是其中一员。

>属于没有自己稳定的内核的人，那就少看这些。

好的！

**Comment by xyzbz** [comment]
我们都在社交媒体茧房里，看多了都麻了。
